.travis.yml

```diff
@@ -1,61 +0,0 @@
-language: generic
-dist: xenial
-services:
-  - xvfb
-
-addons:
-  apt:
-    packages:
-    # Qt dependencies
-    - libglu1-mesa-dev
-    - libxkbcommon-x11-0
-    - libxcb-icccm4
-    - libxcb-image0
-    - libxcb-keysyms1
-    - libxcb-randr0
-    - libxcb-render-util0
-    - libxcb-xinerama0
-    - pulseaudio
-    - libpulse-mainloop-glib0
-    # Wx dependencies
-    - libsdl2-2.0-0
-
-env:
-  global:
-    - INSTALL_EDM_VERSION=3.0.1
-      PYTHONUNBUFFERED="1"
-
-matrix:
-  include:
-    - env: RUNTIME=3.6 TOOLKITS="null"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5 pyside2"
-    - env: RUNTIME=3.6 TOOLKITS="wx"
-  allow_failures:
-    - env: RUNTIME=3.6 TOOLKITS="wx"
```

Comment:
Note testing on wx is now only done for the test-with-pip cron job.
Previously it was run every PR (but was an allowed failure so often failures went completely un-noticed anyway)

Reply:
i'm okay with testing against wx only on cron jobs.